config.go

```diff
@@ -965,6 +968,12 @@ func (c ConfigTLS) LoadTLS() (*tls.Config, error) {
 
 	tlsConfig := &tls.Config{}
 
+	// Skip TLS verification
+	if True(c.SkipVerify) {
+		tlsConfig.InsecureSkipVerify = true
+		return tlsConfig, nil
```

Comment:
Don't return because returning here makes this func ignore all its logic (the code that follows it). Skip-verify is just a flag _to_ the config; it's not the entirety of the config.